lib/src/wait.dart

```diff
@@ -23,7 +23,7 @@ class Wait {
   }) async {
     var timeoutEpochMs =
         DateTime.now().millisecondsSinceEpoch + timeout.inMilliseconds;
-    while (isFalse(await isDone())) {
+    while ((await isDone()).isFalse) {
```

Comment:
Would be better if the parens could be removed.
```suggestion
    while (await isDone().isFalse) {
```

Done in https://github.com/jesims/pouch/pull/13

Reply:
I agree it _would_. Doesn't matter if it's done now or not.